src/playermat/Playmat.ttslua

```diff
@@ -45,8 +45,9 @@ local RESOURCE_COUNTER
 
 -- global variable so it can be reset by the Clean Up Helper
 activeInvestigatorId = "00000"
+local drawButton = false
 
-function onSave() return JSON.encode({zoneID = zoneID, playerColor = PLAYER_COLOR, activeInvestigatorId = activeInvestigatorId}) end
+function onSave() return JSON.encode({zoneID = zoneID, playerColor = PLAYER_COLOR, activeInvestigatorId = activeInvestigatorId, drawButton = drawButton}) end
```

Comment:
This is way too long to be a single line.  I know we're not strictly enforcing it, but 100 characters is generally a good guideline.